.env.example

```diff
@@ -0,0 +1,7 @@
+TIDB_HOST='xxxxxxxx.aws.tidbcloud.com'
+TIDB_PORT='4000'
+TIDB_USER='xxxxxxxxxxx.root'
+TIDB_PASSWORD='xxxxxxx'
+TIDB_DB_NAME='test'
+IS_SERVERLESS='true'
+CA_PATH='/etc/ssl/cert.pem'
```

Comment:
CA file path is different by os. So I think comment it is a example, need user modify it.